app/src/main/java/com/duckduckgo/app/onboarding/ui/page/DefaultBrowserPage.kt

```diff
@@ -70,7 +70,7 @@ class DefaultBrowserPage : OnboardingPageFragment() {
     override fun setUserVisibleHint(isVisibleToUser: Boolean) {
         super.setUserVisibleHint(isVisibleToUser)
         if (isVisibleToUser) {
-            applyStyle()
+            // applyStyle()
```

Comment:
We can remove this commented line and also the implementation of the function that it's not used anymore